pxb/jenkins/pxb80-single-platform-run.yml

```diff
@@ -0,0 +1,224 @@
+          name: XTRABACKUP_TARGET_VERSION
+          trim: false
+    - string:
+        default: 'https://raw.githubusercontent.com/percona/percona-xtrabackup/8.0/storage/innobase/xtrabackup/test/bootstrap.sh'
+        description: 'Pass an URL for downloading bootstrap.sh, If empty will use from repository you specified in PXB80_REPO'
+        name: BOOTSTRAP_URL
+        trim: false
+    - choice:
+        name: XBCLOUD_TYPE
+        choices:
+        - "aws"
+        - "google"
+        description: Choose XBCLOUD type to run the test.
+    - password:
+        default: 'Test@123'
+        description: 'xbcloud access key'
+        name: XBCLOUD_ACCESS_KEY
+        trim: false
+    - password:
+        default: 'Test@123'
+        description: 'xbcloud secret key'
+        name: XBCLOUD_SECRET_KEY
+        trim: false
+    - string:
+          default: ''
+          description: XBCLOUD extra parameters
+          name: XBCLOUD_EXTRA
+          trim: false
+    axes:
+      - axis:
+         type: user-defined
+         name: CMAKE_BUILD_TYPE
+         values:
+          - RelWithDebInfo
+    builders:
+    - shell: |
+          #!/bin/bash -xe
+
+          if [[ "$XBCLOUD_TYPE" == "aws" ]]; then
+              export XBCLOUD_CREDENTIALS="--storage=s3 --s3-access-key=$XBCLOUD_ACCESS_KEY --s3-secret-key=$XBCLOUD_SECRET_KEY $XBCLOUD_EXTRA"
+          else
+              export XBCLOUD_CREDENTIALS="--storage=google --google-access-key=$XBCLOUD_ACCESS_KEY --google-secret-key=$XBCLOUD_SECRET_KEY  $XBCLOUD_EXTRA"
+          fi
+          rm -rf jenkins-pipelines || true
+          git clone https://github.com/Percona-Lab/jenkins-pipelines -b PXB-2363
+          cd jenkins-pipelines
+          if [ -f /usr/bin/yum ]; then
+              sudo -E yum -y erase percona-release || true
+              sudo find /etc/yum.repos.d/ -iname 'percona*' -delete
+              until sudo -E yum -y install epel-release; do
+                  sudo yum clean all
+                  sleep 1
+                  echo "waiting"
+              done
+              until sudo -E yum -y makecache; do
+                  sudo yum clean all
+                  sleep 1
+                  echo "waiting"
+              done
+
+              RHEL=$(rpm --eval %rhel)
+              sudo yum -y install wget
+              PKGLIST="libcurl-devel cmake make gcc gcc-c++ libev-devel openssl-devel"
+              PKGLIST="${PKGLIST} libaio-devel perl-DBD-MySQL vim-common ncurses-devel readline-devel"
+              PKGLIST="${PKGLIST} zlib-devel libgcrypt-devel bison perl-Digest-MD5"
+              PKGLIST="${PKGLIST} socat numactl-libs numactl"
+
+              if [[ ${RHEL} != 8 ]]; then
+                  PKGLIST+=" python-sphinx python-docutils"
+              else
+                  PKGLIST+=" python3-pip python3-setuptools python3-wheel wget ncurses-compat-libs lz4 lz4-devel"
+              fi
+
+              if [[ ${RHEL} -eq 6 ]]; then
```

Comment:
CentOS 6 was deprecated, please remove this IF block

Reply:
done